specs/Appearance.md

```diff
@@ -0,0 +1,85 @@
+Appearance (Theme) API
+===
+
+# Background
+This API's main use is to set the overall appearance for WebView2. The options are similar to Edge: match the system default theme, change to light theme, or change to dark theme. 
+This API has 2 main changes relevant to the end users. First, it sets appearance for WebView2 UI like dialogs, prompts, context menu, etc. And second, this API sets the ['prefers-color-scheme'](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features). Websites typically media query for this setting in order to set CSS themes for light/dark. 
+
+Please note this API will only set the overall appearance, but not theme.
+For reference, in the screenshot below, this API is meant to expose the Overall Appearance section as a WebView2 API. 
+
+![Alt text](.\media\EdgeSettingsAppearance.png "a title")
```

Comment:
Image isn't rendering. Looks like it doesn't like your backslashes. Maybe making them forward slashes will fix?
```suggestion
![Alt text](./media/EdgeSettingsAppearance.png "a title")
```